cmd/livepeer/starter/starter.go

```diff
@@ -290,9 +291,20 @@ func DefaultLivepeerConfig() LivepeerConfig {
 }
 
 func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
-	if *cfg.MaxSessions <= 0 {
-		glog.Fatal("-maxSessions must be greater than zero")
-		return
+	if *cfg.MaxSessions == "auto" && *cfg.Orchestrator {
+		if *cfg.Transcoder {
+			glog.Fatal("-maxSessions 'auto' cannot be used when both -orchestrator and -transcoder are specified")
+			return
+		}
+		core.MaxSessions = 0
+	} else {
+		intMaxSessions, _ := strconv.Atoi(*cfg.MaxSessions)
+		if intMaxSessions <= 0 {
+			glog.Fatal("-maxSessions must be 'auto' or greater than zero")
+			return
+		}
```

Comment:
```suggestion
		intMaxSessions, err := strconv.Atoi(*cfg.MaxSessions)
		if err != nil || intMaxSessions <= 0 {
			glog.Fatal("-maxSessions must be 'auto' or greater than zero")
			return
		}
```
Please handle the err here.